iFrame Widget/Dokumentation/index.html

```diff
@@ -0,0 +1,176 @@
+                </figure>
+            </div>
+            <div class="card" id="embeddingToComponent">
+                <div class="page-header">
+                    <h1>Embedding to Component</h1>
+                </div>
+                <h3>Receive Data from Parent Site via JavaScript</h3>
+                <figure class="highlight">
+                    <pre><code class="language-html" data-lang="html"><span class="functionName">getData()</span> {<br>    <span class="var">const</span> <span class="varName">url</span> = window.location.href;<br>    <span class="var">let</span> <span class="varName">composer</span> = <span class="varName">url</span>.slice(<span class="varName">url</span>.indexOf(<span class="string">'Composer'</span>) + <span class="string">'Composer:'</span>.length);<br>    <span class="var">let</span> <span class="varName">composition</span>;<br>    <span class="varName">composer</span> = <span class="varName">composer</span>.replace(new RegExp(<span class="string">'_'</span>, <span class="string">'g'</span>), <span class="string">' '</span>);<br>    if (<span class="varName">url</span>.indexOf(<span class="string">'Composition'</span>) >= 0) {<br>        <span class="varName">composition</span> = <span class="varName">url</span>.slice(<span class="varName">url</span>.indexOf(<span class="string">'Composition:'</span>) + <span class="string">'Composition:'</span>.length, <span class="varName">url</span>.indexOf(<span class="string">';Composer'</span>));<br>        <span class="varName">composition</span> = <span class="varName">composition</span>.replace(new RegExp(<span class="string">'_'</span>, <span class="string">'g'</span>), <span class="string">' '</span>);<br>    }<br>}</code></pre>
+            </div>
+        </div>
+
+        <!-- jQuery (necessary for Bootstrap's JavaScript plugins) -->
+        <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.12.4/jquery.min.js"></script>
+        <!-- Include all compiled plugins (below), or include individual files as needed -->
+        <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js" integrity="sha384-Tc5IQib027qvyjSMfHjOMaLkfuWVxZxUPnCJA7l2mCWNIpG9mGCD8wGNIcPD7Txa" crossorigin="anonymous"></script>
+        <script>
+            function startUpWidget() {
+                //Transform URL to Information
+                //var url = window.location.href;
+                //Only for Testing!
+                var url = "http://imslp.org/wiki/Category:Beethoven,_Ludwig_van";
+                var url = "http://imslp.org/wiki/Studien_im_Generalbass,_Contrapunkt_und_in_der_Compositionslehre_(Beethoven,_Ludwig_van)";
```

Comment:
Yes, just forgot to comment the other one - was for testing whether Composer or Composition.